customer-satisfaction/materializer/custom_materializer.py

```diff
@@ -58,7 +58,7 @@ def handle_return(
         self,
         obj: Union[
             str,
-            np.ndarray,
+            numpy.ndarray,

```

Comment:
Why this change?

Reply:
I was facing an error with the materializer which didn't make sense! I was just modifying a lot of stuff and didn't revert this back! will do it right now